content/blog/off-label-uses-in-ggplot2/index.Rmd

```diff
@@ -0,0 +1,100 @@
+---
+output: hugodown::hugo_document
+
+slug: off-label-uses-in-ggplot2
+title: Off-label uses in ggplot2
+date: 2021-06-24
+author: Thomas Lin Pedersen
+description: >
+    How can ggplot2 break with no breaking changes? This short post goes into detail with how
+    and why the latest ggplot2 release disrupted some users workflow.
+
+photo:
+  url: https://data-imaginist.com
+  author: Thomas Lin Pedersen
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [programming] 
+tags: [ggplot2, off-label]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
```

Comment:
Did you run this one? I noticed the square thumbnail took a little while to load.

Reply:
Ah - I forgot... thanks